### DISCUSSION TITLE
[cheat] Zelda OoT cheat code

### DISCUSSION THREAD

**Discussion by JPSama77** [project64/project64#1915] [General]
I need help to create a cheat in which I can select the amount of energy (hearts) that i want, without having to have the energy to the maximum.

**Comment by CCrashCup** [comment]
You can edit the "Max -> Heart Containers" cheat to set the number of hearts to what you want. Follow this procedure.
 1. Run PJ64
 2. Select game and play it
 3. Wait at load save game screen
 4. Load the cheat screen
 5. Uncheck the Max -> Heart Containers box
 6. Close cheat screen
 7. Load your current save game
 8. Load the cheat screen
 9. Check the Max -> Heart Containers box
10. On the right, change the \<value\> based on this table:
     Hearts     \<value\>
        1 .....        0010
        2 .....        0020
        3 .....        0030
        4 .....        0040
        5 .....        0050
        6 .....        0060
        7 .....        0070
        8 .....        0080
        9 .....        0090
       10 .....        00A0
       11 .....        00B0
       12 .....        00C0
       13 .....        00D0
       14 .....        00E0
       15 .....        00F0
       16 .....        0100
       17 .....        0110
       18 .....        0120
       19 .....        0130
       20 .....        0140

11. Click "Update Cheat"
12. Exit cheat screen (Number of hearts in HUD should update)
13. Save the game
14. Load the cheat screen
15. Uncheck the Max -> Heart Containers box
16. Set \<value\> back to Max (20) from above table (Steps 10 and 11)
17. Close cheat screen
18. Stop Emulation
19. Start Emulation
20. Load your game again and play.

**Reply by JPSama77** [reply]
Thanks! I think it worked, but after step 18, I can't start the emulation, when I start it the home screen loads but when I want to go to the part of loading my game one of these two warnings appears, I don't know what to do next, sorry for the inconvenience.
![ZELDA](https://user-images.githubusercontent.com/78901228/107709354-adacc600-6c8a-11eb-89ef-479739dee452.JPG)
![ZELDA2](https://user-images.githubusercontent.com/78901228/107709364-b0a7b680-6c8a-11eb-878b-f7ce4505f10a.JPG)

**Reply by CCrashCup** [reply]
I noticed you are using version 2.3.2 of PJ64.
Try using version 2.4.0 from the nightly builds.
Grab the .zip file and unpack it in a location where you have full permissions.
https://www.pj64-emu.com/nightly-builds
